src/selectors/disabled.js

```diff
@@ -91,12 +95,19 @@ export const getNodeDisabled = createSelector(
     focusedModularPipeline,
     visibleSidebarNodes,
     visibleModularPipelineInputsOutputs,
-    disabledModularPipeline
+    disabledModularPipeline,
+    slicedPipeline,
+    isSliceApplied
   ) =>
     arrayToObject(nodeIDs, (id) => {
       let isDisabledViaModularPipeline =
         disabledModularPipeline[nodeModularPipelines[id]];
 
+      let isDisabledViaFilters = false;
+      if (slicedPipeline.length > 0 && isSliceApplied) {
```

Comment:
```suggestion
      if (isSliceApplied && slicedPipeline.length > 0) {
```

Reply:
could you please let me know what happens if getNodeDisabled returns True/False and how can a node be disabled via a slice ? Is this used for a sliced view ?

Reply:
hey i dont think `getNodeDisabled` would return true or false, it will return an object with IDs and its value, eg:

```
{
  1e3cc50a :false
  3b199c6b: false
  3fb71518: true
}
```

so if its false, the node/element on the chart will be faded out. So any nodes in slicedPipeline will have the value of true, and the rest will be false. Does it make sense?